payments-core/src/main/java/com/stripe/android/googlepaylauncher/GooglePayLauncherActivity.kt

```diff
@@ -56,10 +57,7 @@ internal class GooglePayLauncherActivity : AppCompatActivity() {
             return
         }
 
-        val statusColor = intent.getIntExtra(GooglePayLauncherContract.EXTRA_STATUS_BAR_COLOR, -1)
-        if (statusColor != -1) {
-            window.statusBarColor = statusColor
-        }
+        window.statusBarColor = Color.TRANSPARENT
```

Comment:
Can we set this in styles instead?

Reply:
Yep, will do this. This comment also resulted in https://github.com/stripe/stripe-android/pull/6899, so thanks 🙏